app/controllers/PaymentDueController.scala

```diff
@@ -73,7 +73,7 @@ class PaymentDueController @Inject()(val checkSessionTimeout: SessionTimeoutPred
         financialTransactionsService.getAllUnpaidFinancialTransactions.map {
           case transactions if hasFinancialTransactionsError(transactions) => itvcErrorHandler.showInternalServerError()
           case transactions: List[FinancialTransactionsModel] => Ok(views.html.paymentDue(financialTransactions = transactions,
-            paymentEnabled = isEnabled(Payment),backUrl = backUrl, implicitDateFormatter = dateFormatter)).addingToSession(SessionKeys.chargeSummaryBackPage -> "paymentDue")
```

Comment:
Please remove this from the view instead of setting it to true

Reply:
done